bin/migrate-oats-data/documents/post_launch/oats_documents_to_alcs_documents_inquiry.py

```diff
@@ -0,0 +1,189 @@
+from common import (
+    setup_and_get_logger,
+    BATCH_UPLOAD_SIZE,
+    OATS_ETL_USER,
+    add_timezone_and_keep_date_part,
+    OatsToAlcsDocumentSourceCode,
+)
+from db import inject_conn_pool
+from psycopg2.extras import RealDictCursor
+import os
+
+etl_name = "import_inquiry_documents_from_oats"
+logger = setup_and_get_logger(etl_name)
+
+"""
+    This script connects to postgress version of OATS DB and transfers data from OATS documents table to ALCS documents table.
+
+    NOTE:
+    Before performing document import you need to import Inquiries from oats.
+"""
```

Comment:
The same as above, if the comment is under the functions documentation then it will be visible in IDE when referencing the function